jerry-ext/extfunc/extfunc-print.c

```diff
@@ -0,0 +1,112 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript-ext/extfunc.h"
```

Comment:
@zherczeg Please, elaborate. What do you mean by current plans? And what do you mean by unrelated stuff and quickly grow too large? Are there any other common but non-standard functions we'd like to provide implementation for? If they are so non-standard, then they will most probably be app-specific and not useful for many. More importantly, we will not be able to add them to the library if they have _any_ dependency other than jerry-core (and libc+libm, perhaps). Note that even `print` has issues (see proposed implementation here, topmost commit): we cannot even rely on `printf` which is a libc function (because of port limitations)!

As for the path renaming, I'm not sure I get it. `jerry-ext/include` is the dir for public headers. And all `jerry-ext` extensions are to be used by users as `#include "jerryscript-ext/XXX.h"`. This means that all such headers will have to be under `jerry-ext/include/jerryscript-ext` in the source tree. (IIRC there was a discussion about this in #1740)